package.json

```diff
@@ -86,7 +87,7 @@
     "ts-node": "^10.9.1",
     "tsconfig-paths": "^4.2.0",
     "tslib": "2.5.3",
-    "typescript": "4.8.4",
+    "typescript": "4.9.4",
```

Comment:
I also bumped Typescript out but I think I am going to bump it further. I wanted to utilize the `satisfies` keyword introduced in `4.9`.